x/evm/types/hooks.go

```diff
@@ -0,0 +1,96 @@
+package types
+
+import (
+	"math/big"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	bankkeeper "github.com/cosmos/cosmos-sdk/x/bank/keeper"
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	ethcmn "github.com/ethereum/go-ethereum/common"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+)
+
+var (
+	// BankSendEvent represent the signature of
+	// `event __CosmosNativeBankSend(address recipient, uint256 amount, string denom)`
+	BankSendEvent abi.Event
+
+	_ EvmHooks = MultiEvmHooks{}
+	_ EvmHooks = BankSendHook{}
+)
+
+// MultiEvmHooks combine multiple governance hooks, all hook functions are run in array sequence
+type MultiEvmHooks []EvmHooks
```

Comment:
Imo we should also register the module name on MultiEVMHooks for better error messages

Reply:
also let's move this file to the `keeper/` directory

Reply:
done